server.js

```diff
@@ -1,6 +1,8 @@
 const express = require('express');
 const server = express();
 const chalk = require('chalk');
+const { writeFileSync } = require('fs');
+const parser = require('xml2json');
```

Comment:
I've noticed that this library mentions it takes a bit of work getting it installed on Windows. We know that some cavy-cli users use Windows, so I'm mindful of making this a hard dependency of cavy-cli when not everyone needs JUnit XML support.

Could we:

 * Explore other libraries?
 * Make it a `peerDependency`, detect the presence, and guide people to install it if they want JUnit XML support (e.g. indicate the support is disabled because xml2json is not installed).
 * Not worry about it?

Reply:
Good spot! I've had a refreshed look around, and this seems to be pretty popular: https://github.com/Leonidas-from-XIV/node-xml2js.

I've had a look through issues / code / README and there doesn't seem to be any windows specific setup / issues.

Agree that it would be a shame to have a hard dependency when only a subset of people want JUnit! I'm going to swap in this package - should be simple.

Reply:
Works even better! It prettifies and you don't have to convert to JSON before building the XML 👌

Reply:
Very nice